httpbin/httpbin.go

```diff
@@ -93,6 +93,11 @@ type HTTPBin struct {
 	Observer Observer
 }
 
+type bearerResponse struct {
+	Authenticated bool   `json:"authenticated"`
+	Token         string `json:"token"`
+}
```

Comment:
Sorry for the _extreme_ nit pick, but let's move this response type up above the `type HTTPBin struct {}` declaration so that it's grouped with the other response types.

Reply:
Makes sense. Done. I would've done that too :)